src/models/transactions/verify.ts

```diff
@@ -0,0 +1,86 @@
+import { ValidationError } from '../../common/errors'
+import { Transaction } from '.'
+import { isEqual } from 'lodash';
+import { encode, decode } from 'ripple-binary-codec'
+import { verifyAccountDelete,
+         verifyAccountSet, 
+         verifyCheckCancel, 
+         verifyCheckCash, 
+         verifyCheckCreate, 
+         verifyDepositPreauth, 
+         verifyEscrowCancel, 
+         verifyEscrowCreate, 
+         verifyEscrowFinish, 
+         verifyOfferCancel, 
+         verifyOfferCreate, 
+         verifyPaymentChannelClaim, 
+         verifyPaymentChannelCreate, 
+         verifyPaymentChannelFund, 
+         verifyPaymentTransaction, 
+         verifySetRegularKey, 
+         verifySignerListSet, 
+         verifyTicketCreate, 
+         verifyTrustSet } from ".";
+
+export function verify(tx: Transaction) {
```

Comment:
This method should be in `Transaction` and should also be exported at the `src/models/transactions` level

Reply:
Done!